components/Spoiler.tsx

```diff
@@ -0,0 +1,24 @@
+import {Button} from "./Button";
+import {useState} from "react";
+
+type Props = {
+    readonly hiddenText: string
+}
+
+export function Spoiler({ hiddenText }: Props) {
+    const [openBtnText, setOpenBtnText] = useState<boolean>(true);
```

Comment:
Lepší by bylo pojmenování `showText` a `setShowText`, s tlačítkem to tolik nesouvisí.

Reply:
Ještě drobnost - netřeba uvádět typ <boolean>, Typescript to odvodí sám podle typu init hodnoty. Typ je potřeba uvádět jen v případě, že začínáme např. `null` a potom tam dáváme něco jiného.